packages/containers/src/List/List.connect.js

```diff
@@ -6,29 +6,40 @@ import {
 	configureGetFilteredItems,
 	configureGetPagination,
 	configureGetPagedItems,
+	getCollectionItems,
+	configureGetSortedItems,
 } from './selector';
 
 function componentId(ownProps) {
 	return ownProps.collectionId;
 }
 
 function getItems(state, config) {
-	const items = configureGetFilteredItems(config)(state);
-
-	return items || new List();
+	let items = config.items;
+	if (config.defaultFilter !== false) {
+		items = configureGetFilteredItems(config)(state);
+	}
+	if (config.defaultSorting !== false) {
+		items = configureGetSortedItems(config, items)(state);
+	}
+	return items;
 }
 
-function getPagedItems(state, config) {
-	const items = configureGetPagedItems(config)(state);
-
-	return items || new List();
+function getPagedItems(state, config, items) {
+	if (config.defaultFilter !== false && config.defaultSorting !== false) {
```

Comment:
Why don't you add a `props.defaultPage(ing)`. It's the same issue as before, this is too implicit to have default pagging is defaultFilter and defaultSorting is  activated :/

Reply:
fixed